### PR TITLE
Code review: Feature/milestone 4

### PR DESCRIPTION
**Pull request by bafiam**
#### What is this PR about
- Consist of the calculator logic
- Consist of operation logic
#### How to Use

Action keys [`=`, `AC`, `%`, `+/-`]--Double click to execute

`=` - execute the operations

`AC` - clear the inputs all or latest entry

`%` - convert to percentage either first number, second or results

`+/-` - convert to negative or positive either the first number, second or results

#### Live link
[Heroku](https://protected-sierra-43792.herokuapp.com/)

#### Linter errors
- The errors are related the the provided linter config file conflicting with recent `jsx-a11y` updates. To resolve then , a lot of changes will have to be done on the linter json file.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  if (operation === '÷') {
+    if (numberOne === '0' && numberTwo === '0') {
+      return 'Math Error';
+    }
+    if (numberOne !== '0' && numberTwo === '0') {
+      return 'Math Error';
+    }
+
```
Comment (shloch) [inline src/logic/operate.js]: There's math error only if `numberTwo` is zero, if `numberOne` is zero, the answer should be zero

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @bafiam ,

Great job...please fix very minor issue below and also fix the linter error still appearing

### Happy coding

File: src/logic/operate.js [hunk 2]
```diff
@@ -49,9 +49,6 @@ const operate = (numberOne, numberTwo, operation) => {
   }
 
   if (operation === '÷') {
-    if (numberOne === '0' && numberTwo === '0') {
-      return 'Math Error';
-    }
     if (numberOne !== '0' && numberTwo === '0') {
       return 'Math Error';
     }
```
Comment (shloch) [inline src/logic/operate.js]: This is all you need
```
if (operation === '÷') {
    if (numberTwo === '0') {	
      return 'Math Error';	
    }	
   return a.div(b).valueOf();
}
```

**Review by shloch — APPROVED** [review]
You still made a small error.... [ see below ]

Please correct before you merge and move to next milestone

### You get a thumbs up, i count on you

File: src/logic/operate.js (hunk 2 above)
Comment (bafiam) [reply src/logic/operate.js]: omg, didn't see that, thanks for the feedback, working on it now. 

Kind regards,
bafiam

**Pull request merged**